test/web-platform-tests/to-run.yaml

```diff
@@ -848,7 +848,6 @@ execution-timing/074.html: [fail, Unknown]
 execution-timing/076.html: [fail, Unknown]
 execution-timing/077.html: [fail, Unknown]
 execution-timing/078.html: [fail, Unknown]
-execution-timing/079.html: [fail, Unknown]
```

Comment:
This test was failing because it depends on setting `window.location`, which was previously unsupported.